fgbio/1.1.0/CallMolecularConsensusReads.nf

```diff
@@ -0,0 +1,24 @@
+process CallMolecularConsensusReads {
+    tag {"FGBIO_callmolecularconsensusreads ${sample_id}"}
+    label 'FGBIO_1_1_0'
+    label 'FGBIO_callmolecularconsensusreads_1_1_0'
+    clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.callmolecularconsensusreads.mem}" : ""
+
+    // container = 'library://sawibo/default/bioinf-tools:gatk4.1.3.0'
+
+    input:
+      tuple sample_id, file(bam)
+
+    output:
+      tuple sample_id, file("${sample_id}.u.consensus.bam")
+
```

Comment:
Hardcoded paths, should be in params.

Reply:
Totally agree, this is still under heavy development